examples/README.md

```diff
@@ -40,6 +40,11 @@
 You can omit the value of `@Param` if you compiled your code with `-parameters` javac option.
 Please refer to [Configure `-parameters` javac option](http://line.github.io/armeria/setup.html#configure-parameters-javac-option) for more information.
 
+## How to run
+
+ - Use `run` or `bootRun` task to run an example from Gradle.
+ - If you using Intellij IDEA, See also [Build and run the application](https://www.jetbrains.com/help/idea/creating-and-running-your-first-java-application.html#run_app).
```

Comment:
you using -> you're using
See -> see

Reply:
How about:

```markdown
- Use `run` ...
- See [Open an existing Gradle project](https://www.jetbrains.com/help/idea/gradle.html#gradle_import_project_start) to import an example into IntelliJ IDEA.
- See [Configure `-parameters` javac option](http://line.github.io/armeria/setup.html#configure-parameters-javac-option) to configure IntelliJ IDEA.
```

Reply:
> - Use run ...
> - If you using Intellij IDEA, See also [Build and run the application](https://www.jetbrains.com/help/idea/creating-and-running-your-first-java-application.html#run_app).

to

> - Use run ...
> - See [Build and run the application](https://www.jetbrains.com/help/idea/creating-and-running-your-first-java-application.html#run_app) to run example in the editor of the IntelliJ IDEA.

result:
> - Use `run` or `bootRun` task to run an example from Gradle.
> - See [Build and run the application](https://www.jetbrains.com/help/idea/creating-and-running-your-first-java-application.html#run_app) to run example in the editor of the IntelliJ IDEA.
> - See [Open an existing Gradle project](https://www.jetbrains.com/help/idea/gradle.html#gradle_import_project_start) to import an example into IntelliJ IDEA.
> - See [Configure `-parameters` javac option](http://line.github.io/armeria/setup.html#configure-parameters-javac-option) to configure IntelliJ IDEA.

How about remaining the way i mention above? Maybe most knows it already, But i thought it is very convenient way to run each example in IntelliJ IDEA. Also i'll update file as you suggested. Thanks!!

Reply:
Good idea, @Untaek! How about replacing 'run example in the editor of the IntelliJ IDEA' with 'run an example from IntelliJ IDEA'? (Note 'an' and 'the')

Reply:
Oh and 'build and run...' could be the last item in the list because it will work only after you imported and configured the project.

Reply:
Got it!